apps/vr-tests-react-components/src/stories/SearchBox.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { Steps, StoryWright } from 'storywright';
 import { storiesOf } from '@storybook/react';
-import { SearchBox } from '@fluentui/react-search';
+import { SearchBox } from '@fluentui/react-search-preview';
```

Comment:
I didn't think we changed the package name for preview.  Rather we export it from the unstable part of react-components.